src/app/component-library/components/button/button.component.ts

```diff
@@ -24,6 +24,7 @@ export class ButtonComponent {
   @Input() orientation: ORIENTATION = 'left';
   @Input() faIcon: string = '';
   @Input() buttonType: TYPE = 'button';
+  @Input() textColor: { color: string };
```

Comment:
Instead of having the user pass in a a pure CSS style object like this, I think it would be more convenient for the user just to pass in a color directly and the component should wrap that up into a `{color: ...}` object internally.